### DISCUSSION TITLE
Progression Calendar

### DISCUSSION THREAD

**Discussion by Aslyte** [chromiecraft/chromiecraft#6393] [General]
Hello guys.

Do we have a "expansion (progression) calendar" available?

**Comment by heyitsbench** [comment]
> Do we have a "expansion (progression) calendar" available?

Closest you'll find is the [progression page on the website](https://www.chromiecraft.com/en/progression/).

**Reply by Aslyte** [reply]
Yes, but i was hoping to have smth like "TK will be release on May 2024" for example.

**Reply by heyitsbench** [reply]
Releases pretty much only have their date posted when they're the next in line to be released. The date for SSC's release for example wasn't posted until around a month and a half after Karazhan's release.

**Reply by Aslyte** [reply]
Well, lets w8 for it then, thx bro for the replies!
